dev/breeze/src/airflow_breeze/commands/developer_commands.py

```diff
@@ -144,6 +144,11 @@
             ],
         },
     ],
+    "breeze exec": [
```

Comment:
You should also add "exec" to developer commands above as it otherwise is spearated out:

![image](https://user-images.githubusercontent.com/595491/165310286-bdd5e84b-96f4-4570-b9bf-46baa955087d.png)

I also unified all the "commands and parameters to have "." at the end of help.

Reply:
I thinik it should be added right after "exec" (I tried to sort it according to likelihood it's going to be used).